wd_helper.au3

```diff
@@ -975,7 +975,7 @@ EndFunc   ;==>_WD_ElementOptionSelect
 ;                  $sSelectElement - Element ID of <select> element from _WD_FindElement
 ;                  $sCommand       - Action to be performed. Can be one of the following:
 ;                  |DESELECTALL    - Clear all selections
-;                  |MULTISELECT    - Select <option> elements given in 1D array of labels
+;                  |MULTISELECT    - Select <option> elements given in 1D array of labels and deselect all others
```

Comment:
this change here is not true,
take a look here
https://github.com/Danp2/au3WebDriver/blob/12ce520afc11668e5c4bf07bde7fa317a095cc9b/wd_helper.au3#L1056

the `false` on the end is used in JS here:
https://github.com/Danp2/au3WebDriver/blob/12ce520afc11668e5c4bf07bde7fa317a095cc9b/wd_helper.au3#L1035

and here
https://github.com/Danp2/au3WebDriver/blob/12ce520afc11668e5c4bf07bde7fa317a095cc9b/wd_helper.au3#L1023-L1025

Reply:
You said once that multiselect should not deselect other selected items.
And I keep the same way of acting for `MULTISELECT` as it was before.

Only `SINGLESELECT` deselect other selections.

Reply:
Ok... my misunderstanding. What is the logic is having multiSelect and singleSelect behave differently in this regard?

Reply:
If somebody want to single select then it takes assumption that there is no need to multiselect.
This is almost like normal click but here we also check if desired `<option>` element is `enabled` and is `visible`, and additionally you are clicking by `LabelName` just do not must to wory about finding elements.

Reply:
it should be done like mentioned before here:
https://www.autoitscript.com/forum/topic/205553-webdriver-udf-help-support-iii/?do=findComment&comment=1505254

I mean by using `MULTISELECT` with single element in the array.
```autoit
Local $a_Printer[] = ['Save as PDF']
_WD_ElementSelectAction($WD_SESSION, $sSelectElement, "MULTISELECT", $a_Printer)
```

I mean that only `MULTISELECT` and `SELECTALL`  should expect that you are interacting with `multiple` able `<select>` element.

In this resoning only `MULTISELECT` should be able to be able to meet the conditions described in your scenario:

> I was thinking of the scenario where someone wanted to select one _additional_ option without deselecting the rest.

Reply:
That would mean undoing this -- https://github.com/Danp2/au3WebDriver/issues/368#issuecomment-1208732962

Reply:
Ah yes.
I see now your change it here: https://github.com/Danp2/au3WebDriver/commit/9122d4fb2825f9aefa515a9cbc0e96e98fbe2b23

reverted here: ['multiSelect' should be a 1D non-empty array](https://github.com/Danp2/au3WebDriver/pull/367/commits/fbd5c3a97f22cd1f653ec1df007ee430faf0e9f3)